### PR TITLE
Fix/Regenerate the Loader files incorrectly reordered by the scripts.

### PR DESCRIPTION
**Pull request by nrspruit**
The previous script gen accidently reordered the layers/loader files incorrectly and did not put them back.

This fix correctly addresses the invalid reorder to match the updated script generation.

### CODE REVIEW

**Review by rwmcguir — APPROVED** [review]

File: source/layers/tracing/ze_tracing_cb_structs.h
Comment (lisanna-dettwyler) [inline source/layers/tracing/ze_tracing_cb_structs.h:1]: What generates this file?

File: scripts/input.json
Comment (lisanna-dettwyler) [inline scripts/input.json:1]: This file shouldn't even be checked in

File: source/layers/tracing/ze_tracing_cb_structs.h
Comment (nrspruit) [reply source/layers/tracing/ze_tracing_cb_structs.h:1]: https://github.com/oneapi-src/level-zero/blob/master/scripts/templates/tracing/trc_cb_struct.h.makov

File: scripts/input.json
Comment (lisanna-dettwyler) [reply scripts/input.json:1]: It shouldn't be checked in to either repository since it's an artifact, can you just delete it from your branch?

Comment (nrspruit) [reply scripts/input.json:1]: yes, I can just remove it.

Comment (nrspruit) [reply scripts/input.json:1]: Done, removed.

**Review by lisanna-dettwyler — APPROVED** [review]

**Review by rwmcguir — APPROVED** [review]

**Review by aravindksg — APPROVED** [review]

**Pull request merged**